modules/installation-and-upgrade/pages/container-deployment/suma/server-deployment-suma.adoc

```diff
@@ -60,32 +60,32 @@ For more information, see the section on network requirements in xref:installati
 
 // This sections needs to be run by devs. How would a customer use custom storage? How do we set this up with mgradm?
 
-{productname} {productnumber} defines the required persistent storage volumes by default, and these are created during installation by the mgradm tool if they do not already exist.
+{productname} {productnumber} defines the required persistent storage volumes by default.
+These are created during installation by the [command]``mgradm`` tool if they do not already exist.
 
-These volumes are created by default where podman stores its volumes. 
-[systemitem]``Podman`` stores its volumes in [path]``/var/lib/containers/storage/volumes/``.
+These volumes are created in [path]``/var/lib/containers/storage/volumes/``, where [systemitem]``Podman`` stores its volumes by default.
 
 .Recommendations
 [NOTE]
 ====
-You can take advantage of this simplicity of storage by mounting an external storage device on this directory. 
-Since this is where the postgresql database, binary packages for repositories, caches, operating system images, autoinstallation distributions, and configuration files will all be stored, there are three recommendations:
+You can take advantage of this simplicity of storage by mounting an external storage device on this directory.
+Because this is where the {postgresql} database, binary packages for repositories, caches, operating system images, autoinstallation distributions, and configuration files will all be stored, there are three recommendations:
```

Comment:
```suggestion
You can leverage the simplicity of storage by mounting an external storage device to this directory.
Since it will store the {postgresql} database, binary packages for repositories, caches, operating system images, autoinstallation distributions, and configuration files, we have three recommendations:
```